tools/install-deps

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+set -xue
+
+if [[ "$OSTYPE" =~ "linux" ]]; then
+    if which apt-get; then
+        # Debian or Ubuntu
```

Comment:
It is impossible to make a script working on all distributions. It should work on [supported](http://mxe.cc/#requirements) ones.

I tried to use `lsb_release` instead, but it is not installed by default on some OSes, e.g. Gentoo.